web/pages/group/[...slugs]/index.tsx

```diff
@@ -139,6 +147,7 @@ export default function GroupPage(props: {
     topTraders,
     creatorScores,
     topCreators,
+    aboutPost,
```

Comment:
It's good to get `aboutPost` from `getStaticProps`.

However, it might also be nice to listen for changes to `aboutPost` client-side.

One reason to do this is `getStaticProps` is often cached for a period of time so if you saved changes, they might not appear for minutes, even after refreshing.

The solution is to create a hook called `usePost`, which fetches and listens for updates to a post. You can basically copy the implementation of `useGroup`. 

Then in this component, you could use `const aboutPost = usePost(group.aboutPostId) ?? props.aboutPost`.